src/coreclr/src/binder/inc/assembly.hpp

```diff
@@ -148,7 +153,7 @@ namespace BINDER_SPACE
         AssemblyName            *m_pAssemblyName;
```

Comment:
This has a lot of non-DACized pointers. It is hard to see that none of them will get used by the DAC.

Reply:
I have DACized only the ones that are needed for getting the ALC. It was painful enough that way.

Reply:
I understand that it was painful. It is hard to see that it is correct. We prefer the type members to be fully DACized. Partially DACized types are bug farms.